packages/components/src/theme/index.tsx

```diff
@@ -36,16 +28,14 @@ extend( [ namesPlugin, a11yPlugin ] );
  * };
  * ```
  */
-function Theme( props: WordPressComponentProps< ThemeProps, 'div', true > ) {
-	const { accent } = props;
-	if ( accent && ! colord( accent ).isValid() ) {
-		// eslint-disable-next-line no-console
-		console.warn(
-			`wp.components.Theme: "${ accent }" is not a valid color value for the 'accent' prop.`
-		);
-	}
+function Theme( {
+	accent,
+	background,
+	...props
+}: WordPressComponentProps< ThemeProps, 'div', true > ) {
+	const themeVariables = generateThemeVariables( { accent, background } );
```

Comment:
The next thing we should do is to take these `themeVariables` values and make them available to descendant components via Context. This will allow us to resolve the remaining TODO comments in Button, where Button needs to access the color values directly and compute its non-standard colors in JS.

Reply:
Here's a few considerations (almost a brainstorming sessions) that came to mind when thinking about using React Context:

 - are we ok exposing the variables both in React Context and as CSS variables ?
 - Which one is the source of truth?
 - Is there a scenario in which they could not be in sync ?
 - CSS vars will be accessible to DOM descendants, while React Content will be accessible to all react child components — which gives us the chance of using one or the other when styling Slot-Fill components (e.g Popover)